content/english/Kusto-KC7/activity-1.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Activity 1: Intro to Threat Intelligence"
```

Comment:
Since there is no interactive work for the students to do here, we can potentially omit the "Activity" prefix